lib/hydranorth/preservation_queue.rb

```diff
@@ -0,0 +1,17 @@
+module Hydranorth::PreservationQueue
+
+  QUEUE_NAME = YAML.load(File.read('config/preservation.yml'))[Rails.env]['queue_name']
+
+  def queue
+    $queue ||= ConnectionPool.new({size: 1, timeout: 5}) { Redis.new }
```

Comment:
eh...i guess we kinda doing this maybe: https://github.com/ualbertalib/HydraNorth/blob/master/config/initializers/redis_config.rb